### ISSUE TITLE
[WebUI] Filter on profiler token

### ISSUE THREAD

**Issue by rvanlaak** [php-translation/symfony-bundle#67]
Would be great if the web UI would be able to filter on a profiler token. Since 2.7 the profiler collects all translations: https://github.com/symfony/symfony/pull/14003

* allow to filter the shown translations based on a profiler token
* show `<select>` with the last 10 profile tokens
* show button to load the last profile token
* add link in profiler itself that directly filters on the translations of the visited token

Also see https://github.com/lexik/LexikTranslationBundle/issues/165

**Comment by Nyholm** [comment]
I do not think that is a part of the WebUI. The feature is available by the Profiler integration. 

How to enable: http://php-translation.readthedocs.io/en/latest/symfony/profiler-ui.html

Image: https://raw.githubusercontent.com/Happyr/TranslationBundle/master/src/Resources/doc/images/missing-translation-example.gif
Image: https://raw.githubusercontent.com/Happyr/TranslationBundle/master/src/Resources/doc/images/edit-flag-sync-example.gif

**Comment by rvanlaak** [comment]
Yes I know it's not part of the WebUI, but that basically is my feature request; being able to filter based on the profiler tokens. Could be an easy way for developers to do translations; navigate a request, and thereafter "goto last request" in the Web UI.

**Comment by Nyholm** [comment]
Im saying that you _can_ do that already. But you have to use the symfony profiler integration. 

The profiler integration provides that exact functionality and the WebUI is for editing all your translations. 

If one would merge the two we have to duplicate the functionality from Symfony where they show/select profiler tokens and load the data collected.

**Comment by rvanlaak** [comment]
The main reason for this feature request is that the WebUI also shows the translations in the other locales. Yes I agree that functionality is comparable, but I think that the WebUI eventually would be able to give a better translation experience as the profiler would be able to do. 

So it might be a nice discussion to determine what place is preferable for new features, the webUI or the profiler 👍

**Comment by Nyholm** [comment]
> WebUI also shows the translations in the other locales.

Ah, I see. I actually had that problem. I usually go to Loco and translate new strings there. 

👍

**Comment by rvanlaak** [comment]
I'm still in the process of trying to find out whether we would still need an external translation service. As we really prefer translating based on requests, I started submitting feature requests based on that. I hope that we eventually don't need an "extra" piece of software for our translations :)

So, yes I agree that some features duplicate existing behavior (profiler vs. WebUI), but I don't think they would block each other. Eventually from the perspective of the translator this integration between both should work seamlessly.
